docs/man/libnng.3.adoc

```diff
@@ -419,6 +419,7 @@ These functions are intended for use with HTTP server applications.
 |xref:nng_http_server_res_error.3http.adoc[nng_http_server_res_error()]|use HTTP server error page
 |xref:nng_http_server_start.3http.adoc[nng_http_server_start()]|start HTTP server
 |xref:nng_http_server_stop.3http.adoc[nng_http_server_stop()]|stop HTTP server
+|xref:nng_http_server_get_addr.3http.adoc[nng_http_server_get_addr()]|get HTTP server address
```

Comment:
This stanza was alphabetically sorted, so can you move this line up a few to it's correct alphabetical location?